incubator/hnc/cloudbuild.yaml

```diff
@@ -32,6 +32,11 @@ steps:
     # Build plugin
     GOOS=linux GOARCH=amd64 go build -o kubectl-hns_linux_amd64 ../cmd/kubectl/main.go
     GOOS=darwin GOARCH=amd64 go build -o kubectl-hns_darwin_amd64 ../cmd/kubectl/main.go
+    mkdir kubectl
+    cp kubectl-hns_linux_amd64 kubectl/
+    cp kubectl-hns_darwin_amd64 kubectl/
+    cp ../LICENSE kubectl/
+    tar czvf kubectl-hns.tar.gz kubectl/
```

Comment:
Why didn't see this part in #1136 but here?

Reply:
In v0.6, cloudbuild.yaml defers to the makefile for almost everything. In v0.5 it replicates what's in the makefile, so I had to do the same.